### PR TITLE
normalized my naming pattern to bem methodology

### PR DESCRIPTION
**Pull request by mbukeRepo**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -39,7 +40,7 @@ p {
 header {
```
Comment (Jonath-z) [inline styles.css:40]: avoid HTML tag

**Review by Jonath-z — CHANGES_REQUESTED** [review]
kindly apply this review

File: styles.css (hunk 1 above)
Comment (mbukeRepo) [reply styles.css]: Thanks @Jonath-z  I've tried to apply your review.

**Review by Jonath-z — APPROVED** [review]
LGTM

File: index.html [hunk 2]
```diff
@@ -86,7 +86,7 @@ <h1>Meet Our Experts</h1>
         <div class="teams-section__desc">
```
Comment (IbrahimBagalwa) [inline index.html:86]: this is the same issues man you are using `.teams-section__desc`

**Review by IbrahimBagalwa — COMMENTED** [review]
Great job man keep it up :)

File: index.html (hunk 2 above)
Comment (mbukeRepo) [reply index.html:86]: thanks @IbrahimBagalwa for your review, will do that.

**Pull request merged**